README.md

```diff
@@ -7,6 +7,8 @@ or simpler alternatives.
 This plugin complements [babel-plugin-lodash](https://www.npmjs.com/package/babel-plugin-lodash)
 by shrinking its cherry-picked builds even further!
 
+DISCLAIMER: Using this plugin without enabling the proper feature sets will cause lodash functions to behave in unexpected ways. Calling the methods will still work, however they might return incorrect results.
```

Comment:
`will cause` -> `can cause`.

`Calling the methods will still work` -> `Methods may appear to work, however`.